pkg/operator/flags.go

```diff
@@ -0,0 +1,38 @@
+package operator
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+const (
+	AlertWebhookEnabled                = "aro.alertwebhook.enabled"
+	AzureSubnetsEnabled                = "aro.azuresubnets.enabled"
+	AzureSubnetsNsgManaged             = "aro.azuresubnets.nsg.managed"
+	AzureSubnetsServiceEndpointManaged = "aro.azuresubnets.serviceendpoint.managed"
+	BannerEnabled                      = "aro.banner.enabled"
+	CheckerEnabled                     = "aro.checker.enabled"
+	DnsMasqEnabled                     = "aro.dnsmasq.enabled"
```

Comment:
nit: Dnsmasq isn't camelCase. It's technically `dnsmasq` but Go wants exported things to start with a capital letter.

Reply:
I've changed it both for `DnsmasqEnabled` and `RestardDnsmasqEnabled`, thanks for pointing out :)